libsolidity/inlineasm/AsmAnalysis.cpp

```diff
@@ -411,3 +413,21 @@ Scope& AsmAnalyzer::scope(Block const* _block)
 	solAssert(scopePtr, "Scope requested but not present.");
 	return *scopePtr;
 }
+
+void AsmAnalyzer::warnOnFutureInstruction(solidity::Instruction _instr, SourceLocation const& _location)
+{
+	switch (_instr)
+	{
+	case solidity::Instruction::RETURNDATASIZE:
+	case solidity::Instruction::RETURNDATACOPY:
+		m_errors.push_back(make_shared<Error>(
+			Error::Type::Warning,
+			"The RETURNDATASIZE/RETURNDATACOPY instructions are only available after "
```

Comment:
@chriseth @pirapira this is the main question in this PR though: what wording do we go with 😉